.gitignore

```diff
@@ -48,3 +48,7 @@ doc/*/doxygen-output
 
 # IDE metadata
 nbproject/
+
+# Compilation database, as may be generated by tools like Bear
```

Comment:
Huh, had never heard of Bear before. TIL

Reply:
Another case where GitHub's selection of only 8 emojis for reactions feels woefully inadequate. :bear:

Reply:
```
   ,__ ., __, ,,,,
   '--/,,\--'\*\%\*\
     //  \\\'\'%.\'%\
      '..'//'%\.\%/\\'.^
         \\'/'/%''/\'
           ||     ||
           "      "
```